README.md

```diff
@@ -254,7 +253,7 @@ See [LICENSE-APACHE](LICENSE-APACHE), [LICENSE-MIT](LICENSE-MIT), and
 
 ## Trademark
 
-The Rust programming language is an open source, community project governed
```

Comment:
These are both correct, I think. I don't see any reason to change it.

Reply:
Going off the wiki, I was under the assumption `open-source` was correct.

https://en.wikipedia.org/wiki/Open-source_software

Reply:
And there's also https://en.wikipedia.org/wiki/Open_source. I don't think one is preferred over the other.

Reply:
When used as an adjective before a noun, use "open-source" (with a hyphen) in your writing to help prevent confusion. For example, in the sentence "I enjoy using open-source software." the word "open-source" describes a type of software.

When used as a noun, use "open source" (with no hyphen). For example, in the sentence "My software is open source." the use of "open source" is not an attributive adjective and does not require a hyphen.

[0](https://www.merriam-webster.com/dictionary/open-source)

Reply:
It should be `open-source` because it is describing the project.

When used as an adjective before a noun, use "open-source" (with a hyphen) to prevent confusion. For example, in the sentence "I enjoy using open-source software." the word "open-source" describes a type of software.

When used as a noun, use "open source" (with no hyphen). For example, in the sentence "My software is open source." the use of "open source" is not an attributive adjective and does not require a hyphen.

[0](https://www.merriam-webster.com/dictionary/open-sourcel)

Reply:
https://opensource.org/ uses "Open Source" exclusively.